public/views/login-step2.ejs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+
+<h2>2-steps login form</h2>
+
+<form action="<%= baseUrl %>/secure/login" method="post">
+    <input type="hidden" name="_csrf" value="<%= csrfToken %>">
+    <input type="hidden" name="username" value="<%= username %>">
```

Comment:
Thought there was an xss here, but good to know that ejs does escaping on it's own 💪🏻